### PR TITLE
.NET SDK templates

### LINKED ISSUE #1: [Android] `dotnet new android-fsharp` should create a copy of net6.0-android-fsharp
Following this documentation: https://docs.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-project-template

We need to package `net6.0-android-fsharp` as a template compatible with `dotnet new`.
Ideally, we will be able to do: `dotnet new android-fsharp -n MyAndroidApp`

### PR DESCRIPTION
**Pull request by gsomix**
Templates:
- [x] Android 
- [ ] iOS
- [ ] Mac Catalyst
- [ ] MAUI
- [ ] Xamarin.Forms

Packaging docs used: https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates#packing-a-template-into-a-nuget-package-nupkg-file

Closes #1 #2 #3

### CODE REVIEW

**Comment by gsomix** [comment]
@TimLariviere I've started to draft .NET SDK templates and need your preliminary review. What do you think about changing structure of repository and naming of folders/projects?

File: Mobile.FSharp.Templates.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>0.0.1</PackageVersion>
+    <PackageId>Mobile.FSharp.Templates</PackageId>
```
Comment (TimLariviere) [inline Mobile.FSharp.Templates.csproj]: I would name it `FSharp.Mobile.Templates` to follow the same convention than `FSharp.Android.Resource`

Comment (TimLariviere) [reply Mobile.FSharp.Templates.csproj]: But yes, I like the renaming. We can even add the missing old F# templates to the package.

Comment (gsomix) [reply Mobile.FSharp.Templates.csproj]: Thanks! I'll continue to work on templates in near time.

Comment (TimLariviere) [reply Mobile.FSharp.Templates.csproj]: Thanks! I'll merge this one for now

**Pull request merged**